espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/GetSessions.java

```diff
@@ -0,0 +1,26 @@
+package io.appium.espressoserver.lib.handlers;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
+import io.appium.espressoserver.lib.model.AppiumParams;
+import io.appium.espressoserver.lib.model.Session;
+
+public class GetSessions implements RequestHandler<AppiumParams, Collection<String>> {
```

Comment:
Do we want to return sorted or unsorted sequence?
If sorted then it's better to use List interface for the returned type.